tslint.json

```diff
@@ -9,10 +9,9 @@
     "no-shadowed-variable": false,
     "no-string-literal": false,
     "trailing-comma": [false],
-    "member-ordering": [true, "fields-first"],
     "array-type": [true, "array"],
     "arrow-parens": false,
-    "max-classes-per-file": [false],
+    "max-classes-per-file": false,
     "prefer-for-of": false,
     "no-implicit-dependencies": false
```

Comment:
I think in this "PR", this file should not be modified. Because, even if it needs to be modified, it has nothing to do with the subject.